label_studio/users/templates/users/user_account.html

```diff
@@ -176,9 +176,9 @@
   <script>
     (() => {
       {% if settings.HOSTNAME %}
```

Comment:
Hi @Ben-Epstein. Thanks for the fix!

What do you think think of adding another javascript const ?

```suggestion
      {% if settings.HOSTNAME %}
      const hostname = {{ settings.HOSTNAME }};
      {% else %}
      const hostname = 'http://localhost:8080';
      {% endif %}
```

Reply:
hey, sure! Works for me, and i think this is cleaner.